openshift/operator-controller.Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN make go-build-local
 FROM registry.ci.openshift.org/ocp/4.19:base-rhel9
 USER 1001
 COPY --from=builder /build/bin/operator-controller /operator-controller
-COPY openshift/manifests /openshift/manifests
+COPY openshift/operator-controller/manifests /openshift/operator-controller/manifests
```

Comment:
The manifest destination should remain at `/openshift/manifests`. Only the source _should_ change.

Reply:
Done Good catcher